tests/compose/test-misc-tweaks.sh

```diff
@@ -66,8 +73,13 @@ cp -a /usr/etc/foo.txt /usr/share/misc-tweaks-foo.txt
 EOF
 chmod a+x postprocess.sh
 
+# Check that the `setup` package contains the files `/etc/csh.login` and `/etc/csh.cshrc` 
```

Comment:
Actually, would it make more sense to build a fake package with `build_rpm` to test this? This way, we're not at the mercy of updates to the `setup` package.

Reply:
:+1:  I think that's a good idea. Could probably combine it with the new `barbar` package. So e.g. you could change the filter for `barbar` to `/usr/bin/.*` and `/etc/sharedfile` and check that `/usr/bin/barbar` is gone, but not `/etc/sharedfile`.